### PR TITLE
sys/transceiver: Replace eINT/dINT

### PR DESCRIPTION
**Pull request by gebart**
Replaced `dINT`/`eINT` pairs with `disableIRQ()`/`restoreIRQ()`.

Quite straightforward change.

### CODE REVIEW

File: sys/transceiver/transceiver.c [hunk 1]
```diff
@@ -672,7 +672,7 @@ void receive_nativenet_packet(radio_packet_t *trans_p)
 void receive_at86rf231_packet(ieee802154_packet_t *trans_p)
 {
     DEBUG("Handling AT86RF231 packet\n");
-    dINT();
+    unsigned int mask = disableIRQ();
```
Comment (LudwigOrtmann) [inline sys/transceiver/transceiver.c]: for consistency, how about changing these to `unsigned state` or adopting the pattern for existing places (at least `receive_nativenet_packet`) ?

**Comment by gebart** [comment]
Adapted according to comments

**Comment by LudwigOrtmann** [comment]
ACK, please squash

**Comment by gebart** [comment]
squashed

**Comment by OlegHahm** [comment]
like this?

**Comment by OlegHahm** [comment]
obsoleted

**Pull request closed without merging**